src/main/java/seedu/address/logic/commands/TagDeleteCommand.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Deletes a tag identified using it's displayed index from the tag list.
+ */
+public class TagDeleteCommand extends Command {
```

Comment:
Can I check why are there two TagDeleteCommand.java?

Reply:
Oops sorry my bad, deleted one of them.